src/Nethermind/Nethermind.Optimism/OptimismHeaderValidator.cs

```diff
@@ -73,8 +84,37 @@ protected override bool ValidateRequestsHash(BlockHeader header, IReleaseSpec sp
 
     protected override bool ValidateGasLimitRange(BlockHeader header, BlockHeader parent, IReleaseSpec spec, ref string? error) => true;
 
+    protected override bool ValidateGasUsed(BlockHeader header, ref string? error)
+    {
+        if (!base.ValidateGasUsed(header, ref error))
+            return false;
+
+        if (specHelper.IsJovian(header))
+        {
+            if (header.BlobGasUsed is not { } blobGasUsed)
+            {
+                if (_logger.IsWarn) _logger.Warn($"Invalid block header ({header.Hash}) - no DA footprint in {nameof(header.BlobGasUsed)}");
+                error = ErrorMessages.DaFootprintMissing;
+                return false;
+            }
+
+            if ((long)blobGasUsed > header.GasLimit)
+            {
+                if (_logger.IsWarn) _logger.Warn($"Invalid block header ({header.Hash}) - gas used above gas limit");
+                error = ErrorMessages.DaFootprintExceededGasLimit;
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    protected override ulong? CalculateExcessBlobGas(BlockHeader parent, IReleaseSpec spec) => 0;
```

Comment:
Why is this here? There should be no calculations performed in validation.

Reply:
There are no calculations, it return's 0. This way I'm reusing `base.ValidateBlobGasFields` to validate that `ExcessBlobGas` is 0. There are other useful (and required) checks performed in this method, and I dind't want to copy it.